pkg/apis/kubernetes/v1/integration_test.go

```diff
@@ -0,0 +1,6 @@
+//go:build integration
+// +build integration
+
+package v1
+
+// TODO: implement integration tests
```

Comment:
the fun thing with your mock tests: change the client to go to the real API and you have an integration test :)